src/openscad.cc

```diff
@@ -784,6 +784,16 @@ int gui(const vector<string> &inputFiles, const fs::path &original_path, int arg
 }
 #endif // OPENSCAD_QTGUI
 
+std::pair<string, string> customSyntax(const string& s)
+{
+#if defined(Q_OS_MACX)
+	if (s.find("-psn_") == 0)
+		return make_pair(string("psn"), s.substr(5));
```

Comment:
Tiny suggestion: `return {"psn", s.substr(5)};` should do the job here.

Reply:
I didn't actually know what `make_pair` did, I was just following the example. I assumed it was some kind of special struct. 😆

Reply:
`make_pair()` simply constructs a pair using ADL (argument-dependent lookup). C++11 is capable of initializing this directly based on the return type.